pkg/devfile/parser/data/interface.go

```diff
@@ -35,4 +35,8 @@ type DevfileData interface {
 	GetCommands() []common.DevfileCommand
 	AddCommands(commands []common.DevfileCommand) error
 	UpdateCommand(command common.DevfileCommand)
+
+	AddVolume(volume common.Volume, path string) error
+	DeleteVolume(name string) error
+	GetVolumeMountPath(name string) (string, error)
```

Comment:
does this function needs be here in interface? it is looking like a util one.

Reply:
v3 can have a different schema and design, so I think we can keep it here.